emulator/src/test/java/com/scalar/client/tool/emulator/Callee.java

```diff
@@ -25,10 +25,19 @@
 import java.util.Optional;
 import javax.json.Json;
 import javax.json.JsonObject;
+import javax.json.JsonObjectBuilder;
 
 public class Callee extends Contract {
   @Override
   public JsonObject invoke(Ledger ledger, JsonObject argument, Optional<JsonObject> properties) {
-    return Json.createObjectBuilder().add("callee_is_called", true).build();
+    JsonObjectBuilder result = Json.createObjectBuilder();
+    if (getCertificateKey() != null) {
```

Comment:
In a real contract, it never returns null, so it might be better to set the default certificate as I suggested above.